packages/playground/src/dashboard/components/public_config.vue

```diff
@@ -77,30 +82,39 @@
                 ]"
                 #="{ props }"
               >
-                <input-tooltip tooltip="Gateway for the IP in ipv6 format">
-                  <v-text-field v-model="config.gw6" v-bind:="props" outlined label="Gateway IPv6"></v-text-field>
+                <input-tooltip tooltip="Gateway for the IP in IPv6 format">
+                  <v-text-field v-model="config.gw6" v-bind="props" outlined label="Gateway IPv6"></v-text-field>
                 </input-tooltip>
               </input-validator>
 
+              <!-- Domain -->
               <input-validator
                 :value="config.domain"
                 :rules="[domain => validators.isURL('Wrong domain format.')(domain)]"
                 #="{ props }"
               >
                 <input-tooltip tooltip="Domain for web gateway">
-                  <v-text-field v-model="config.domain" v-bind:="props" outlined label="Domain"></v-text-field>
+                  <v-text-field v-model="config.domain" v-bind="props" outlined label="Domain"></v-text-field>
                 </input-tooltip>
               </input-validator>
             </form-validator>
           </div>
           <v-card-actions class="justify-space-between px-5 pb-5 pt-0">
+            <!-- Remove and Generate Config Buttons -->
             <v-btn
-              @click="showClearDialogue = true"
+              @click="isNodeHasConfig"
               color="error"
               variant="outlined"
               :disabled="isRemoving || Object.values(config).every(value => value == '')"
-              >Remove Config</v-btn
             >
+              Remove Config
+            </v-btn>
+
+            <v-btn v-if="runningInDevMode" @click="fillFakeData" color="warning" variant="outlined">
```

Comment:
do we need this btn ?

Reply:
Yes it helps us in development mode to fill fake data, don't worry it won't appear in the production, only development